CSS/firstjump.css

```diff
@@ -0,0 +1,180 @@
+body {
+  background-color: white;
+  padding: 0px;
+  margin: 0px;
+  width: 100%;
+}
+
+/* Font section */
+
+.site_font {
+  font-family: "Roboto Condensed", sans-serif;
+}
+
+.site_font.bold {
+  font-weight: 700;
+}
+
+.site_font.light {
+  font-weight: 300;
+}
+
+.site_font.regular {
+  font-weight: 400;
+}
+
+.site_alt_font_one {
+  font-family: cursive;
+  font-size: 24px;
+}
+
+.site_alt_font_two {
+  font-size: 24px;
+}
+
+/* Banner */
+
+#bannertitle {
+  font-size: 7vw;
+  text-align: center;
+  color: white;
+  background-color: #4282d6;
+}
+
+.banner_title {
+  margin-top: 0%;
+  background-color: #4282d6;
+  padding-top: 10%;
+}
+
+#bannerImageContainer {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  position: relative;
+  background-color: #4282d6;
+}
+
+#bannerClouds {
+  width: 100%;
+  margin-top: 30%;
+  background-size: cover;
+  overflow: hidden;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  position: relative;
+}
+
+#bannerSkydiver {
+  width: 80%;
+  position: absolute;
+  margin-left: 12%;
+  margin-bottom: 5%;
+}
+
+#infoSection {
+  margin-top: -2%;
```

Comment:
Almost always if youre doing a negative percent youre doing it wrong and gonna cause other issues (almost always). This could be accomplished without it.